Add support for more widgets and improve activation

rwx788 opened the pull request:
See https://github.com/libyui/libyui-rest-api/pull/5

Commit:
Add qt specific action handler for rest-api

Instead of adding activation methods to every widget we want to support,
we now can trigger events we need from the override directly, so no
change in underlying libyui libraries will be required.

Commit:
Trigger update action on selection in combobox

Commit:
Add activation for YSelectionBox and restructure the code

After moving one control, we can see that we can easily avoid
duplication and only override activation actions if they require
special handling.

Commit:
Add activation for YDate and YTime fields

Commit:
Add activation for YInputField

Commit:
Add activation for YMultiSelectionBox

lslezak commented on README.md:
```diff
@@ -4,5 +4,8 @@ These are the [libyui REST API](https://github.com/libyui/libyui-rest-api)
 bindings for the [libyui-qt](https://github.com/libyui/libyui-ncurses-rest-api)
 UI frontend.
 
+This package contains qt specific handling of the actions. Library is dynamically
```
Qt themselves call `Qt`, with upper case `Q` :smiley:

lslezak reviewed (approved):
Just a small spelling suggestion...

Commit:
Add support for YCheckBoxFrame widget

With QT there is a deviation to behavior we see when use some of the
examples manually (e.g. CheckBoxFrame3.rb), but for the other cases
it works just fine. I will check if I can improve this code, otherwise
we can enable and disable all the children widget to be able to test
functional part ignoring UI interaction.

Commit:
Update README file

Commit:
Bump version to 0.2.0 and update changelog
